test/backend/routes.spec.js

```diff
@@ -122,292 +51,6 @@ function closeServers() {
 
 // Suppressing ESLint cause Mocha ensures `this` is bound in test functions
 /* eslint-disable no-invalid-this */
-describe('Servers:', () => {
```

Comment:
I see you have removed these tests, but I cant find where you added them..

Reply:
Excellent catch!